wiki/Tournaments/Official_support/en.md

```diff
@@ -271,6 +273,8 @@ Please consult the following examples of acceptable banner designs:
 
 ![](https://assets.ppy.sh/main-menu/egts-2021/gf.png)
 
+Use [this banner visualisation tool](https://tcomm.hivie.tn/assets-previewer?tab=in-game-banners) to check how the banner would look in-game.
```

Comment:
```suggestion
Use [this banner visualisation tool](https://tcomm.hivie.tn/assets-previewer?tab=in-game-banners) to check how the banner would look in game.
```
This is not a compound adjective, so it doesn't need the hyphen.

Reply:
it is something that has always been spelt with a hyphen on the wiki regardless 🤔
https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#terminology

Reply:
would prefer to stick to in-game unless we change our terminology for this